test/sharness/t0053-dag.sh

```diff
@@ -133,48 +179,47 @@ test_dag_cmd() {
   '
 
   test_expect_success "output looks correct" '
-    echo "{\"data\":\"CAISBGZvbwoYBA==\",\"links\":[]}" > cat_exp &&
+    echo -n "{\"Data\":{\"/\":{\"bytes\":\"CAISBGZvbwoYBA==\"}},\"Links\":[]}" > cat_exp &&
```

Comment:
Did we intend to change capitalization here or is this accidental?

This seems like it may need a changelog entry.  I can't say for sure how much people might use whatever API this is, but I suspect it might be nonzero?

Reply:
Yes, changed to match the data model form for DAG-PB: https://ipld.io/specs/codecs/dag-pb/spec/#logical-format

This is now consistent across Go & JS (long-standing problem) and matches the pb schema too. And also DAG-JSON with its bytes format making a showing here too now—although this test has padding it in and shouldn't since we've fixed that @ https://github.com/ipld/go-ipld-prime/pull/216 so I'll have to do another pass here.